UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FssService.cs

```diff
@@ -115,5 +123,221 @@ public async Task<Stream> DownloadFile(string downloadPath, string fileName, str
                 throw new FulfilmentException(EventIds.DownloadFileFailed.ToEventId());
             }
         }
+
+        public async Task<string> CreateBatch(string mediaType)
```

Comment:
This method is covered by one test which checks for an invalid http response (see DoesCreateBatch_Throws_Exception_If_InValidRequest) but there is no test covering a valid http reponse (i.e. lines 138 to 142). Should there be a test checking a valid response?

Reply:
Added test case for valid create batch scenario.